Gaijinizer patch/Map471.txt

```diff
@@ -3,1663 +3,2153 @@
+up on their own.#RonIffy
 > END STRING
 
 > BEGIN STRING
 でも、キミにしちゃ浅はかだったと思うよ
 
 例えるなら、バグに対処するのが怖くなって
 強制的にパソコン切っちゃうような行為に近いし……
-> CONTEXT: Map471/events/1/pages/0/744/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map471/events/1/pages/0/744/Dialogue/sirousagi_fc1
+But I think doing that to you was short-sighted.
+It's like getting scared of a bug in a computer
+program so you forcibly cut the power to the
+whole PC.#Ron
 > END STRING
 
 > BEGIN STRING
 そのうえ、\\C[2]\\N[1]くんに
 情が湧いちゃってる可能性を考えずに接触しちゃうなんてさ
 
 \\C[0]そりゃ、アバドーンにも目を付けられちゃうよ
-> CONTEXT: Map471/events/1/pages/0/749/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map471/events/1/pages/0/749/Dialogue/sirousagi_fc1
+On top of that, \\C[2]going for that kind of approach
+ran the risk of causing certain emotions to well
+up within \\N[1]-kun, \\C[0]and also attracted
+the attention of Abaddon.#Ron
 > END STRING
 
 > BEGIN STRING
 ………返す言葉もないよ。
 『手先』に寝返る介入者が、あんなに多いとは思わなくて…
 
 実際、状況はそれで更に悪化してしまったからね……
-> CONTEXT: Map471/events/1/pages/0/754/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map471/events/1/pages/0/754/Dialogue/enemy27
+You're completely right. I didn't think so many
+of the Interveners had double-crossed us to be
+its pawns, and it actually made the situation
+even worse.#Ron
 > END STRING
 
 > BEGIN STRING
 僕という\\C[10]『介入媒体』\\C[0]が
 実体化しているのを発見したアバドーンは、標的を変更した。
 
 そう……\\C[2]僕にだ。
-> CONTEXT: Map471/events/1/pages/0/762/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map471/events/1/pages/0/762/Dialogue/enemy27
+Abaddon had detected the manifestation of my
+\\C[10]Intervention medium\\C[0], and decided to change
+targets... \\C[2]to me.#Ron
 > END STRING
 
 > BEGIN STRING
 ヤツは\\C[2]僕を喰い、自分の\\C[10]介入能力と融合\\C[2]させる気なんだ。
 
 \\C[0]そして……\\C[4]真慈悲\\C[0]の『正史』の介入権限を奪い
 直接、\\C[10]正史全体を改変してしまう。
-> CONTEXT: Map471/events/1/pages/0/767/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map471/events/1/pages/0/767/Dialogue/enemy27
+The plan was \\C[2]to consume me and fuse with my
+Intervention abilities\\C[0]. Doing so would provide
+Abaddon with the authority to intervene in the
+True Mercy of the official timeline.
+From there, \\C[10]he could make wholesale changes as
+he pleased.#Ron
 > END STRING
 
 > BEGIN STRING
 待て待て！サラッと言うな！！
 
 \\C[10]正史を改変\\C[0]だと！？
 いきなりヤバさの規模がでかくなったぞ！？
-> CONTEXT: Map471/events/1/pages/0/772/Dialogue/alice_fc5 < UNTRANSLATED
+> CONTEXT: Map471/events/1/pages/0/772/Dialogue/alice_fc5
+Hold on, hold on! Don't just breeze over that!
 
+\\C[10]Make wholesale changes\\C[10]?! The stakes suddenly
+got a whole lot higher!!#Ron
 > END STRING
 
 > BEGIN STRING
 だから言ってるじゃん、ヤバいんだって……
 
 しかも、マッドハッターの\\C[10]『介入媒体』\\C[0]は
 アバドーンが持ってる半端なものとはワケが違う。
-> CONTEXT: Map471/events/1/pages/0/777/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map471/events/1/pages/0/777/Dialogue/sirousagi_fc1
+That's what we've been saying. Even worse, the
+\\C[10]Intervention medium \\C[0]Mad Hatter has is different
+to the half-baked one Abaddon has.#Ron
 > END STRING
```

Comment:
Rewording:
```
That's what we've been saying...  Even worse
is that Mad Hatter, the \\C[10]Intervention
Medium,\\C[0] is different then the half-baked
one Abaddon has.
```

I'm making the assumption here that Mad Hatter literally *is* the Intervention Medium, which is the impression I got. I'm not quite sure if this fits, though?